pkg/kubelet/apis/pluginregistration/v1alpha/constants.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pluginregistration
+
+const (
+	CSIPlugin      = "CSIPlugin"
+	DevicePlugin   = "DevicePlugin"
+	PluginsSockDir = "/var/lib/kubelet/plugins/"
```

Comment:
I remember a discussion where we agreed that it would be better to have
- `/var/lib/kubelet/device-plugin/plugins/`
- `/var/lib/kubelet/csi/plugins/`

So that we could store checkpointing in a different place then the plugins (i.e: /var/lib/kubelet/device-plugin/)
Also it seems a bit weird to have a common dir for CSO and device plugin how would that be handled?